### ISSUE TITLE
some questions about the dp_mask

### ISSUE THREAD

**Issue by JullyZ** [facebookresearch/DensePose#109]
hi，
   i've run your project, and i found that the 'AnnIndex''s shape is (7,25,56,56). I think the AnnIndex may action as dp_mask。dp_mask has 14 parts，so I think the AnnIndex may be 15 channel。Why here it has 25 channels. I see the network in your project, i found the  AnnIndex is parallel with the IUV branch，but in your paper, you introduce a cross-cascading architecture. you don't use this architecture in your code?

**Comment by PedroCastro** [comment]
That is true, they do not use it in the published code. Confirmed by one of the authors [here](https://github.com/facebookresearch/DensePose/issues/62#issuecomment-404228774).
